### PR TITLE
OrderEdit-Action-Examples

### PR DESCRIPTION
**Pull request by czacherl**

### CODE REVIEW

File: examples/OrderEdit/OrderEditAddStagedActionAction.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "action": "addStagedAction",
+  "stagedAction": [
+    "stagedOrderUpdateAction"
```
Comment (jenschude) [inline examples/OrderEdit/OrderEditAddStagedActionAction.json]: Please either change to 
```
"stagedAction": [
]
```
or one valid staged example action e.g.:
```
"stagedAction": [
  {
     "action": "setComment",
     "comment": "commentString"
  }
]
```

File: examples/OrderEdit/OrderEditSetStagedActionsAction.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "action": "setStagedActions",
+  "stagedActions": "stagedOrderUpdateActionString"
```
Comment (jenschude) [inline examples/OrderEdit/OrderEditSetStagedActionsAction.json]: See comment at addStagedAction example. The structure should be a valid example.

File: examples/OrderEdit/StagedOrderSetLocaleAction.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "action": "setLocale",
+  "locale": "de-de"
```
Comment (jenschude) [inline examples/OrderEdit/StagedOrderSetLocaleAction.json]: Please change to a valid locale e.g. `de-DE`

File: examples/OrderEdit/StagedOrderTransitionStateAction.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "action": "transitionState",
+  "state":{
+    "typeId": "state",
+    "id": "{{stateId}}"
+  },
+  "force": true
```
Comment (jenschude) [inline examples/OrderEdit/StagedOrderTransitionStateAction.json]: i would remove the `force` property. We should not promote force for transitioning states

File: examples/OrderEdit/OrderEditAddStagedActionAction.json (hunk 1 above)
Comment (czacherl) [reply examples/OrderEdit/OrderEditAddStagedActionAction.json]: example added

File: examples/OrderEdit/StagedOrderSetLocaleAction.json (hunk 3 above)
Comment (czacherl) [reply examples/OrderEdit/StagedOrderSetLocaleAction.json]: done

File: examples/OrderEdit/StagedOrderTransitionStateAction.json (hunk 4 above)
Comment (czacherl) [reply examples/OrderEdit/StagedOrderTransitionStateAction.json]: removed

File: examples/OrderEdit/OrderEditAddStagedActionAction.json (hunk 1 above)
Comment (jenschude) [reply examples/OrderEdit/OrderEditAddStagedActionAction.json]: I'm so sorry. `setComment` is no stageable action. Could you please change it so some action with can be staged?

File: examples/OrderEdit/OrderEditSetStagedActionsAction.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "action": "setStagedActions",
+  "stagedActions": [
+    "recalculate"
```
Comment (jenschude) [inline examples/OrderEdit/OrderEditSetStagedActionsAction.json]: Still no valid action. Should be
```
"stagedActions": [
   {
      "action": "recalculate"
   }
]
```

But please check first if this action is stageable.

File: examples/OrderEdit/OrderEditAddStagedActionAction.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
+  "action": "addStagedAction",
+  "stagedAction": [
+    {
+      "action": "setComment",
```
Comment (jenschude) [inline examples/OrderEdit/OrderEditAddStagedActionAction.json]: I'm so sorry. setComment is no stageable action. Could you please change it so some action with can be staged? Also I see that the structure only allows one action. So it must be:

```
"stagedAction": {
   "action": "..."
}
```

File: examples/OrderEdit/OrderEditSetStagedActionsAction.json (hunk 5 above)
Comment (czacherl) [reply examples/OrderEdit/OrderEditSetStagedActionsAction.json]: switched to setCountry

File: examples/OrderEdit/OrderEditAddStagedActionAction.json (hunk 6 above)
Comment (czacherl) [reply examples/OrderEdit/OrderEditAddStagedActionAction.json]: switched to setCountry

**Pull request merged**